macros/convertToClusterEvents.C

```diff
@@ -0,0 +1,131 @@
+#include <TTree.h>
+#include <TFile.h>
+#include <TH1.h>
+#include <TH2.h>
+#include <Rtypes.h>
+#include <TCanvas.h>
+
+#include <cstdlib>
+#include <iostream>
+#include <iomanip>
+#include <unistd.h>
+
+#include "CTSEvent.h"
+#include "CTSEventClusters.h"
+#include "Utility.h"
+
+///< usage: ./convertToClusterEvents -i inputfile -o outputfile -n numberOfEventsToBeProcessed
+///< n = -1 by default which means the whole file is processed
+
+extern char* optarg;
+
+void convertToClusterEvents(const char *inputFile, const char *outputFile, ULong_t procNr)
```

Comment:
To make macro names clearer I would rename this and rather call it `convertToCTSEventClusters`. If at any point we implement another type of event (e.g. physical event) we get confused.